packages/perseus/src/widgets/cs-program/score-cs-program.ts

```diff
@@ -1,9 +1,7 @@
 import type {PerseusScore} from "../../types";
 import type {PerseusCSProgramUserInput} from "../../validation.types";
 
-export function csProgramValidator(
-    state: PerseusCSProgramUserInput,
-): PerseusScore {
+export function scoreCSProgram(state: PerseusCSProgramUserInput): PerseusScore {
```

Comment:
Looks like this scoring file doesn't do the export default at the bottom. Should I update it to export like the others?

Reply:
Yes, please. We mix between `export default` and exporting named functions in Perseus, but we should at least be consistent across the scoring function/files.

Reply:
Jeremy and I disagree; I think `export function` is better and he advocates for having it at the end. I think it's best to be consistent more than anything.